server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfiguration.java

```diff
@@ -53,7 +53,7 @@ public Builder sizeThresholdInBytes(int sizeThresholdInBytes) {
         public Builder ttl(Duration ttl) {
             Preconditions.checkNotNull(ttl, "'TTL' must not to be null");
             Preconditions.checkArgument(ttl.getSeconds() > 0, "'TTL' needs to be positive");
-            Preconditions.checkArgument(ttl.getSeconds() < Integer.MAX_VALUE, "'TTL' must not grater than %s", Integer.MAX_VALUE);
+            Preconditions.checkArgument(ttl.getSeconds() < Integer.MAX_VALUE, "'TTL' must not greater than %s", Integer.MAX_VALUE);
```

Comment:
'TTL' must not `be` greater than

Reply:
error message should not use integer but duration

Reply:
agree

Reply:
What about having a limit for humans, like: 10 years for example?